web/src/components/CompareSide.vue

```diff
@@ -36,6 +40,7 @@ import "prismjs/themes/prism.css";
 import "prismjs/plugins/line-numbers/prism-line-numbers.js";
 import "prismjs/plugins/line-numbers/prism-line-numbers.css";
 import { ID_START, registerFragmentHighlighting } from "@/util/OccurenceHighlight";
+import { NodeStats } from "@dodona/dolos-lib/dist/lib/analyze/SemanticAnalyzer";
```

Comment:
```suggestion
import { NodeStats } from "@dodona/dolos-lib";
```
And `NodeStats` will probably need to be exported by the lib.